### PR TITLE
fix(comments): localize all strings

### PR DESCRIPTION
**Pull request by ninaandal**
### Description
This PR adds a resource file to the comments plugin with all the strings and localizes every string in the comments plugin. 

### What to review
All strings for the comments plugin feature 

### Notes for release
Adds localization to the comments plugin

### CODE REVIEW

File: packages/sanity/playwright-ct/tests/comments/CommentInputStory.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ export function CommentsInputStory({
     <TestWrapper schemaTypes={SCHEMA_TYPES}>
       <CommentInput
         focusOnMount
```
Comment (jtpetty) [inline packages/sanity/playwright-ct/tests/comments/CommentInputStory.tsx:48]: any reason to no localize this?

File: packages/sanity/src/structure/comments/plugin/field/CommentsFieldButton.tsx [hunk 2]
```diff
@@ -185,9 +189,17 @@ export function CommentsFieldButton(props: CommentsFieldButtonProps) {
   }
 
   return (
-    <Tooltip portal placement="top" content={`View comment${count > 1 ? 's' : ''}`}>
+    <Tooltip
+      portal
+      placement="top"
```
Comment (jtpetty) [inline packages/sanity/src/structure/comments/plugin/field/CommentsFieldButton.tsx:194]: There is actually native support for singular/plural built into the i18n library. Here is an example: https://github.com/sanity-io/sanity/blob/ab0dbe1b29c766a3a771ca3c066b9af00a0cd52e/packages/sanity/src/core/form/inputs/files/common/DropMessage.tsx#L31

File: packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx [hunk 3]
```diff
@@ -297,7 +299,7 @@ export function CommentsListItemLayout(props: CommentsListItemLayoutProps) {
 
                   {formattedLastEditAt && (
                     <TimeText muted size={0} title={formattedLastEditAt}>
-                      (edited)
+                      {`(${t('comments.list-item-layout-edited')})`}
```
Comment (jtpetty) [inline packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx:302]: Would 
```
({t('comments.list-item-layout-edited')})
```
be more readable here?

File: packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx [hunk 4]
```diff
@@ -32,10 +33,11 @@ interface UserDisplayNameProps {
 function UserDisplayName(props: UserDisplayNameProps) {
   const {currentUserId, isFirst, userId, separator} = props
   const [user] = useUser(userId)
+  const {t} = useTranslation(commentsLocaleNamespace)
 
   const isCurrentUser = currentUserId === userId
   const you = isFirst ? 'You' : 'you'
-  const content = isCurrentUser ? you : user?.displayName ?? 'Unknown user'
+  const content = isCurrentUser ? you : user?.displayName ?? t('comments.reaction-unknown-user')
```
Comment (pedrobonamin) [inline packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx:40]: Could we add the translation to `you` here?

File: packages/sanity/playwright-ct/tests/comments/CommentInputStory.tsx (hunk 1 above)
Comment (ninaandal) [reply packages/sanity/playwright-ct/tests/comments/CommentInputStory.tsx:48]: We are not localising stories 🙂

File: packages/sanity/src/structure/comments/plugin/field/CommentsFieldButton.tsx (hunk 2 above)
Comment (ninaandal) [reply packages/sanity/src/structure/comments/plugin/field/CommentsFieldButton.tsx:194]: Following the underscore example as provided in our docs for this, just overlooked this, thanks for pointing out 👍

File: packages/sanity/src/structure/comments/plugin/inspector/CommentsInspector.tsx [hunk 5]
```diff
@@ -119,18 +121,18 @@ function CommentsInspectorInner(props: DocumentInspectorProps) {
           pushToast({
             closable: true,
             status: 'info',
-            title: 'Copied link to clipboard',
+            title: t('comments.inspector-copy-title'),
```
Comment (hermanwikner) [inline packages/sanity/src/structure/comments/plugin/inspector/CommentsInspector.tsx:124]: Should we be more specific here and name the keys to e.g:

- `comments.copy-link-success-message`
- `comments.copy-link-error-message`

File: packages/sanity/src/structure/comments/i18n/resources.ts [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+  /**The button tooltip aria label for adding a reaction */
+  'comments.list-item-context-menu-add-reaction-aria-label': 'Add reaction',
+  /**The action menu item for copying a comment link */
+  'comments.list-item-copy-link': 'Copy link to comment',
+  /**The action menu item for deleting a comment */
+  'comments.list-item-delete-comment': 'Delete comment',
+  /**The action menu item for editing a comment */
+  'comments.list-item-edit-comment': 'Edit comment',
+  /**The marker to indicate that a comment has been edited in brackets */
+  'comments.list-item-layout-edited': 'edited',
```
Comment (hermanwikner) [inline packages/sanity/src/structure/comments/i18n/resources.ts:89]: I am not sure if we need to use "layout" in these keys?

File: packages/sanity/src/structure/comments/i18n/resources.ts [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+  /**The button text for confirming discard */
+  'comments.discard-button-confirm': 'Discard',
+  /**The header for discard comment dialog */
+  'comments.discard-header': 'Discard comment?',
+  /**The text for discard comment dialog */
+  'comments.discard-text': 'Do you want to discard the comment?',
+
+  /**The document inspector title for comments */
+  'comments.document-inspector-title': 'Comments',
```
Comment (hermanwikner) [inline packages/sanity/src/structure/comments/i18n/resources.ts:41]: What's the difference between this one and `comments.header-title`?

File: packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx (hunk 4 above)
Comment (ninaandal) [reply packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx:40]: Nice catch!

File: packages/sanity/src/structure/comments/src/components/pte/blocks/MentionInlineBlock.tsx [hunk 8]
```diff
@@ -33,7 +34,9 @@ export function MentionInlineBlock(props: MentionInlineBlockProps) {
   const {selected, userId} = props
   const [user, loading] = useUser(userId)
   const currentUser = useCurrentUser()
+  const {t} = useTranslation(commentsLocaleNamespace)
 
+  // eslint-disable-next-line i18next/no-literal-string
   if (!user || loading) return <Span>@Loading</Span> // todo: improve
```
[the remark this answers is not in the document]
Comment (ninaandal) [reply packages/sanity/src/structure/comments/src/components/pte/blocks/MentionInlineBlock.tsx:40]: Had a chat with Herman about this - I'll create a separate ticket and pr for this instead

**Review by pedrobonamin — APPROVED** [review]

**Review by jtpetty — APPROVED** [review]
👍

**Pull request merged**